src/app/cube/details/components/action-panel/action-panel.component.ts

```diff
@@ -199,10 +203,10 @@ export class ActionPanelComponent implements OnInit, OnDestroy {
    * @param learningObjectId the unique mongo id of a learning object
    */
 
-  download(learningObjectId: string) {
-    this.toggleDownloadModal(true);
-    this.libraryService.downloadBundle(BUNDLING_ROUTES.DOWNLOAD_BUNDLE(learningObjectId));
-  }
+  // download(learningObjectId: string) {
+  //   this.toggleDownloadModal(true);
+  //   this.libraryService.downloadBundle(BUNDLING_ROUTES.DOWNLOAD_BUNDLE(learningObjectId));
+  // }
```

Comment:
We need this. And this.libraryService should be this.bundlingService after the migration to bundling.service.ts.